_includes/02-image.md

```diff
@@ -0,0 +1 @@
+Replace this with an image, like your photo. Ensure you include some alt-text.
```

Comment:
![Image of a Gibson SG Jr](https://user-images.githubusercontent.com/72891396/113439256-d5c5c500-93af-11eb-9764-5edfa3787ccd.jpg)